plugin/session_buffer.py

```diff
@@ -231,7 +235,7 @@ def present() -> None:
             if delay_in_seconds <= 0.0:
                 present()
```

Comment:
Shouldn't we also cancel here?

Reply:
We cancel earlier for all the branches so no? Or what do you mean? :)

Reply:
You're right, I did not notice the above cancellation above.